migrations/108_exam_times_create.sql

```diff
@@ -0,0 +1,7 @@
+/* From PS exam_times as of 2018-02-24 */
+
+CREATE TABLE IF NOT EXISTS exam_times (
```

Comment:
We'll also need to create a file called `database/tables/exam_times.pg` that contains the schema for this new table.